website/views.py

```diff
@@ -88,6 +88,8 @@ def serialize_node_summary(node, auth, primary=True, show_path=False):
             'api_url': node.api_url,
             'title': node.title,
             'category': node.category,
+            'isPreprint': node.is_preprint,
```

Comment:
This will need to check if the file has a preprint id instead, `is_preprint` becomes false if the preprint is made private, which can cause problems.